### DISCUSSION TITLE
Standardization of card layouts with front end filtering

### DISCUSSION THREAD

**Discussion by mcomptonrh** [RedHat-UX/red-hat-design-system#1289] [Ideas]
There are a number of places the UX Studio team has attempted to solve, and is currently trying to solve, similar design challenges related to front end filtering of cards. Unlike our faceted search pages, these examples aren't linking to a particular Drupal content type. We're trying to standardize as much as possible, but thought it would be a good idea if this team could take some time to consider from a design system perspective. 

In flight projects: 
  - https://www.sketch.com/s/a6cd7be5-1523-43a5-ae7f-8eb47d60e83a/a/r9ojYo7
  - https://xd.adobe.com/view/d942fb1d-c3b3-4088-8bc2-a44f20c56ad7-8c11/screen/e1efb38a-8da0-494c-a3b4-8f9112690396
Live experiences:
  - https://www.redhat.com/en/technically-speaking
  - https://www.redhat.com/en/open-source-stories
Brand Media Subsystem topics and taxonomy page (Chaebin Park can provide access): https://xd.adobe.com/view/f59a4c22-c45b-4218-ad4b-e7674c748b3e-a0c9/screen/0cdab990-dc15-45e9-8567-2a80d7ee8420
Future opportunities: 
  - https://www.redhat.com/en/technologies/cloud-computing/openshift/aws/learn
  - https://www.redhat.com/en/interactive-labs/enterprise-linux

**Comment by erob54** [comment]
@mcomptonrh You might be interested in some of the work we recently did on the Developers site - https://developers.redhat.com/learn
